invenio_jsonschemas/ext.py

```diff
@@ -165,6 +166,14 @@ def path_to_url(self, path):
         ).build(
             'schema', values={'path': path}, force_external=True)
 
+    @cached_property
+    def loader(self):
+        """Loader used by `JsonRef.replace_refs`."""
+        loader_cls = self.app.config['JSONSCHEMAS_LOADER']
+        if isinstance(loader_cls, six.string_types):
+            return import_string(loader_cls)
```

Comment:
Ok. I have changed it back to a class and renamed the cached property to `loader_cls` and instantiate it always in the view.